opendevin/agent.py

```diff
@@ -89,17 +82,7 @@ def history(self) -> List[str]:
         return self._history
 
     @abstractmethod
-    def add_event(self, event: Event) -> None:
-        """
-        Adds an event to the agent's history.
-
-        Parameters:
-        - event (Event): The event to add to the history.
-        """
-        pass
-
-    @abstractmethod
-    def step(self, cmd_mgr: CommandManager) -> Event:
+    def step(self, state: State) -> Action:
```

Comment:
"state --- agent ---> action" is a commonly used paradigm in RL i think, conforming to this convention make help people understand what's going on here easily.